gradle/versions.gradle

```diff
@@ -9,29 +9,8 @@ ext.versions = [
 	'xtext_bootstrap': '2.16.0',
 	'gradle_plugins': '0.1.0',
 	'xtext_gradle_plugin': '2.0.2',
-	// When adjusting the `lsp4j` version here, do not forget to do the followings:
-	// - Update the version in `IdeProjectDescriptor#pom`.
-	// - Update the test expectations by running `CliWizardIntegrationTest#main`.
-	// - Update the versions in the `/org.eclipse.xtext.ide/META-INF/MANIFEST.MF` and `/org.eclipse.xtext.ide.tests/META-INF/MANIFEST.MF` files.
-	'lsp4j': '0.6.0',
-	'log4j': '1.2.16',
-	'equinoxCommon' : '3.9.0',
-	'equinoxRegistry' : '3.7.0',
-	'eclipseCore': '3.13.0',
-	'eclipseOSGI' : '3.12.100',
-	'emfCommon': '2.12.0',
-	'emfEcore': '2.12.0',
-	'emfChange': '2.11.0',
-	'emfCodegen': '2.11.0',
-	'emfMwe': '1.3.21.201705291010',
-	'emfMwe2': '2.9.1.201705291010',
+	'dependency_management_plugin' : '1.0.6.RELEASE',
+	'args4j': '2.33',
 	'jdtCore': '3.13.102',
```

Comment:
is this still needed. afaik we no longer use the workaround for signature problem?